src/test/java/pageobject_model/page/CloudGoogleCalculatorPage.java

```diff
@@ -0,0 +1,162 @@
+package pageobject_model.page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import pageobject_model.test.Helpers;
+
+
+public class CloudGoogleCalculatorPage extends BasePage {
+
+    @FindBy(xpath = "//a[@href='https://cloud.google.com/pricing/calculators']")
+    private WebElement calculatorsTab;
+
+    @FindBy(xpath = "//input[@id='input_52']")
+    private WebElement instancesNumberField;
+
+    @FindBy(xpath = "//md-select[@id='select_75']")
+    private WebElement machineTypeDropdown;
+
+    @FindBy(xpath = "//md-option[@value='CP-COMPUTEENGINE-VMIMAGE-N1-STANDARD-8']")
+    private WebElement machineTypeDropdownElement;
+
+    @FindBy(css = "md-input-container > [aria-label='Add GPUs']")
+    private WebElement addGpuCheckbox;
+
+    @FindBy(xpath = "//input[@id='input_52']")
+    private WebElement instancesNumberFieldText;
+
+    @FindBy(xpath = "//md-select[@placeholder='Number of GPUs']/md-select-value")
+    private WebElement gpuNumberDropdown;
+
+    @FindBy(xpath = "//*[@class='md-select-menu-container md-active md-clickable']//md-option[@value='1']")
```

Comment:
1. If you have long list of classes for element, it will be better to write 'contains' logic.
2.  It's better to avoid of locators like: "//*" - because all elements applicable for this locator. Instead of it you could directly search expected element.
Example: 
//div[contains(@class,'md-select-menu-container')]//md-option[@value='1']